.eslintrc

```diff
@@ -28,7 +28,21 @@
     "no-plusplus": ["error", { "allowForLoopAfterthoughts": true }],
     "no-underscore-dangle": "off",
     "no-use-before-define": ["error", { "functions": false, "classes": false }],
-    "import/no-extraneous-dependencies": ["error", { "devDependencies": ["scripts/**", "rollup.config.js", "**/_demo*.js", "**/_story*.js", "**/src/**/*.react.js", "**/demo/**"] }],
+    "import/no-extraneous-dependencies": [
+      "error",
+      {
+        "devDependencies": [
+          "scripts/**",
+          "rollup.config.js",
+          "**/_demo*.js",
+          "**/story.js",
+          "**/_story*.js",
+          "**/*.story.js",
```

Comment:
We have to version for the naming for 'stories'. This is a temp fix that works for all the existing versions. This will be cleaned up in a separate PR.

Reply:
agree